### ISSUE TITLE
[Question] Does FairMOT support TorchScript?

### ISSUE THREAD

**Issue by ievbu** [ifzhang/FairMOT#197]
I'm interested in loading FairMOT using TorchScript scripting/tracing. Is this possible?

**Comment by Ursanon** [comment]
I think it can be hard 'cause `DCNv2` without TorchScript support is used as backbone.
Any news?

**Comment by austinmw** [comment]
@Ursanon that's true, but it may be easier if using a resnet backbone. Though there will still probably be other issues

**Comment by blackCmd** [comment]
> I'm interested in loading FairMOT using TorchScript scripting/tracing. Is this possible?

Yes, I want it too!

**Comment by amancodeblast** [comment]
Any news for the same? Even without the DCNv2 if the conversion can take place with less accuracy it would be fine.
